web/public/locales/sv.json

```diff
@@ -1,4 +1,18 @@
 {
+  "info-modal": {
+    "title": "Info",
+    "intro-text": "Appen electricityMap ger insikter i realtid på webben och mobil för medborgare, energiexperter, icke-statliga organisationer och beslutsfattare och främjar en djupare förståelse för utmaningar i energiomställningen.",
+    "open-source-text": "Den här appen är <a href=\"%s\" target=\"_blank\">Open Source</a> (<a href=\"%s\" target=\"_blank\">se källkoden</a>) och drivs av en community.",
+    "feedback-button": "Ge återkoppling",
+    "github-button": "Se vår Github",
```

Comment:
```suggestion
    "github-button": "Se vår GitHub",
```

Reply:
It says Github in en.json, but I agree with you that it should be GitHub

Reply:
If you want you can update it there too otherwise I'll just do that later.